modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -78,6 +78,12 @@ public PulsarContainer withTransactions() {
         return this;
     }
 
+    @Override
+    public PulsarContainer withStartupTimeout(Duration startupTimeout) {
+        this.startupTimeout = startupTimeout;
+        return super.withStartupTimeout(startupTimeout);
+    }
```

Comment:
We can omit this method and already set the `WaitAllStrategy` in the constructor. The `withStartupTimeout` method of `GenericContainer` will work as expected.